.github/workflows/main.yml

```
@@ -7,13 +7,23 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+        - name: Windows
+          os: windows-latest
+          mill: 'mill --no-server'
+        - name: Linux
+          os: ubuntu-latest
+          mill: 'mill'
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
```

I think it could be good to check the action runs for both Windows and Linux images, but `mill --no-server` shouldn't be required now (see https://github.com/com-lihaoyi/mill/issues/781#issuecomment-1152431216)

```suggestion
    strategy:
      matrix:
        include:
        - name: Windows
          os: windows-latest
        - name: Linux
          os: ubuntu-latest
    name: ${{ matrix.name }}
    runs-on: ${{ matrix.os }}
```

---

Sure that sounds good. I don't think this will be merged anytime soon though - this PR has been outstanding for years.